content-security-policy/connect-src/connect-src-websocket-blocked.sub.html

```diff
@@ -17,16 +17,12 @@
             log("violated-directive=" + e.violatedDirective);
         });
 
-        try {
-            var ws = new WebSocket("ws://{{domains[www1]}}:{{ports[http][0]}}/echo");
+        var ws = new WebSocket("ws://{{domains[www1]}}:{{ports[http][0]}}/echo");
```

Comment:
I think this test is now correct (matching Chromium) but both WebKit and Firefox fail this now so worth double checking.

Reply:
I would expect an event here too. It's not clear why the network layer would synchronously callback.

Reply:
Done

Reply:
Should we also attempt to assert the state here and above?